common/utils.js

```diff
@@ -754,6 +771,7 @@
                 element.on('$destroy', function() {
                     unbindWatch();
                     if (focusPromise) {
+                        // TODO: Attach .catch w/ catchAll
                         $timeout.cancel(focusPromise);
                     }
                 });
```

Comment:
[For line 799](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/utils.js#L799), we want to wrap both functions.